production/ksonnet/loki/config.libsonnet

```diff
@@ -12,16 +12,21 @@
 
     grpc_server_max_msg_size: 100 << 20,  // 100MB
 
-    wal_enabled: true,
     query_scheduler_enabled: false,
     overrides_exporter_enabled: false,
 
-    // flags for running ingesters/queriers as a statefulset instead of deployment type.
-    // WAL enabled configurations automatically use statefulsets.
-    stateful_ingesters: false,
+    // flags for running queriers as a statefulset instead of deployment type.
```

Comment:
I think this comment should go in line 30 next to `stateful_queriers: false,`

Reply:
Removed the comment completely as it does not make sense any more.